scraper.js

```diff
@@ -127,64 +125,35 @@ Scraper.prototype.vratVsechnyNabidky = function () {
 /**
  * Metoda starající se o stahování dat z určitého zdroje.
  *
- * @param idZdroje
- * @returns {Promise} OKOK
- */
-Scraper.prototype.stahniNabidku = function (idZdroje) {
-  var self = this;
-
-  if (offline) {
-    return self.nactiNabidkuZeSouboru(idZdroje);
-  }
-
-  return new Promise(function (resolve, reject) {
-    request({
-        url      : zdroje[idZdroje].url,
-        encoding : null
-      },
-      function (error, response, body) {
-
-        let mojeId = idZdroje;
-        let mujZdroj = zdroje[idZdroje];
-
-        if (error) {
-          reject('nepodařilo se stáhnout nabídku pro ' + mujZdroj.name, error);
-        } else {
-          self[mujZdroj.parser](body, mojeId);
-          resolve(self.nabidka[mojeId]);
-        }
-      }
-    );
-  });
-};
-
-
-
-/**
- * Načítá nabídku z testovacího souboru na disku, podle zdroje.
+ * Pokud je nastavená ENV proměnná OFFLINE_NABIDKA, pak načítá nabídku z
+ * testovacího souboru na disku, podle zdroje.
  *
  * @param idZdroje
- * @returns {Promise} OKOK
+ * @returns {Promise}
  */
-Scraper.prototype.nactiNabidkuZeSouboru = function (idZdroje) {
+Scraper.prototype.stahniNabidku = function (idZdroje) {
   var self = this;
 
   return new Promise(function (resolve, reject) {
-
-    fs.readFile(
-      zdroje[idZdroje].file,
-      function (error, body) {
-        let mojeId = idZdroje;
-        let mujZdroj = zdroje[idZdroje];
-        if (error) {
-          reject('nepodařilo se načíst soubor s nabídkou pro ' + mujZdroj.name, error);
-        } else {
-          self[mujZdroj.parser](body, mojeId);
-          resolve(self.nabidka[mojeId]);
-        }
+    var zpracujZiskanouNabidku = function (err, body) {
+      if (err) {
+        reject('nepodařilo se stáhnout nabídku pro ' + zdroje[idZdroje].name, err);
+      } else {
+        self[zdroje[idZdroje].parser](body, idZdroje);
+        resolve(self.nabidka[idZdroje]);
       }
-    );
-
+    };
+    if (process.env.OFFLINE_NABIDKA == 'TRUE') {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/ondra/obed-o-bot/file/2229873907/issues/source?bid=3165659&fileBranchId=3175486#l146)